index.xml

```diff
@@ -11172,14 +11168,9 @@ partial interface Window {
               <li>Let <var>object</var> be a new object created as if by the expression <code>({})</code>.</li>
               <li>For each of the aforementioned <a class='dfnref' href='#dfn-interface-member'>interface members</a>
                 declared with the <a class='xattr' href='#Unscopable'>[Unscopable]</a> extended attribute,
-                call the <span class='prop'>[[DefineOwnProperty]]</span> method of <var>object</var>
-                passing the <a class='dfnref' href='#dfn-identifier'>identifier</a> of the
-                interface member as the property name, Property Descriptor
-                <span class='descriptor'>{ [[Value]]: <span class='esvalue'>true</span>,
-                  [[Writable]]: <span class='esvalue'>true</span>,
-                  [[Enumerable]]: <span class='esvalue'>true</span>,
-                  [[Configurable]]: <span class='esvalue'>true</span> }</span>
-                and <span class='esvalue'>false</span>.</li>
+                call <a>CreateDataProperty</a>(<var>object</var>,
+                the <a class='dfnref' href='#dfn-identifier'>identifier</a> of the
+                interface member, <span class='esvalue'>true</span>).</li>
```

Comment:
Bit funny to have an English phrase in the argument list here, but I guess it's OK.